examples/basics/visuals/axially_symmetric_surfaces.py

```diff
@@ -0,0 +1,81 @@
+import numpy as np
```

Comment:
I have added docstring in my copy of the project. Should I include license info as in other examples

Reply:
Added license info also